packages/test/test-end-to-end-tests/src/test/data-virtualization/blobHandles.spec.ts

```diff
@@ -0,0 +1,144 @@
+/*!
```

Comment:
Since it's not DV-related, move to a different file?

Reply:
100%